colossus/colossus-htb.js

```diff
@@ -175,9 +175,12 @@ function ColossusHtb(configs) {
          * returned from gdpr.getConsent() are safe defaults and no attempt has been
          * made by the wrapper to contact a Consent Management Platform.
          */
-        queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
-        queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
-
+        if (ComplianceService) {
+            queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
+            queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
+            queryObj.ccpa = ComplianceService.usp && ComplianceService.usp.getConsent();
+        }
+        
```

Comment:
Hi,

Lint test is still failing on this line.
`183:1  error  Trailing spaces not allowed  no-trailing-spaces`
Can you please remove the trailing white spaces?

Thanks!

Reply:
done